raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -442,7 +446,7 @@ def test_events_for_secretreveal_all_states():
         assert search_for_item(
             events,
             SendSecretReveal,
-            {"secret": UNIT_SECRET, "recipient": setup.channels.partner_address(0)},
+            {"secret": UNIT_SECRET, "recipient": setup.channels.partner_privatekeys[0]},
```

Comment:
Im pretty sure this is incorrect. Are you now passing the private key instead of the public address? Why?